scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -386,6 +388,73 @@ def build_and_test_host_config(test_root,host_config):
     sexe("ls %s/include" % install_dir, echo=True, error_prefix="WARNING:")
     sexe("ls %s/lib" %     install_dir, echo=True, error_prefix="WARNING:")
     sexe("ls %s/bin" %     install_dir, echo=True, error_prefix="WARNING:")
+
+    # test the installation using installed cmake examples
```

Comment:
Note: Our LLNL CI tests are now running the cmake and blt examples to verify a proper installation.